packages-exp/auth-compat-exp/src/auth.ts

```diff
@@ -41,16 +41,16 @@ export class Auth
   implements compat.FirebaseAuth, Wrapper<exp.Auth>, _FirebaseService {
   // private readonly auth: impl.AuthImpl;
 
-  constructor(readonly app: FirebaseApp, private readonly auth: exp.AuthImpl) {
+  constructor(readonly app: FirebaseApp, readonly _delegate: exp.AuthImpl) {
```

Comment:
We also have "user.ts" which has an underlying exp class as well. Should that also follow this pattern? `updateProfile(user, {})` vs `user.updateProfile({})` for example

Reply:
Nevermind, that seems irrelevant for this PR since this is specifically for `_FirebaseService`